docs: make the readme more user-friendly and clean (#40)
Enchoseon/2based2wait: README.md

<div align="center">

> 	<h2>2Based2Wait</h1>

Author: epic fail

	<p>Lightweight & (theoretically) extensible 2b2t proxy.</p>
</div>

> <div align="center">
>   <a href="https://github.com/Enchoseon/2based2wait/wiki">Wiki</a>
>   <span> · </span>
>   <a href="https://github.com/Enchoseon/2based2wait/issues/new">Report a bug</a>
>   <span> · </span>
>   <a href="https://github.com/Enchoseon/2based2wait/blob/main/docs/configuration-guide.md">Configuration guide</a>
>   <span> · </span>
>   <a href="https://github.com/Enchoseon/2based2wait/discussions">Discussions</a>
>   <br>
> 	<img src="https://img.shields.io/github/last-commit/Enchoseon/2based2wait?color=2A0944&labelColor=525E75&style=flat" alt="Last Commit">
> 	<img src="https://img.shields.io/github/languages/code-size/Enchoseon/2based2wait?color=3FA796&labelColor=525E75&style=flat" alt="Code Size">
> 	<img src="https://img.shields.io/github/package-json/v/Enchoseon/2based2wait?color=FEC260&labelColor=525E75&style=flat" alt="Current Version">
> 	<img src="https://img.shields.io/github/license/Enchoseon/2based2wait?color=A10035&labelColor=525E75&style=flat" alt="License">
> </div>

Reviewer: ```suggestion

<div align="center">
	<strong>
	<a href="https://github.com/Enchoseon/2based2wait/wiki">Wiki</a>
	<span> · </span>
	<a href="https://github.com/Enchoseon/2based2wait/issues/new">Report a bug</a>
	<span> · </span>
	<a href="https://github.com/Enchoseon/2based2wait/blob/main/docs/configuration-guide.md">Configuration guide</a>
	<span> · </span>
	<a href="https://github.com/Enchoseon/2based2wait/discussions">Discussions</a>
	</strong>
	<br><br>
	<img src="https://img.shields.io/github/last-commit/Enchoseon/2based2wait?color=2A0944&labelColor=525E75&style=flat" alt="Last Commit">
	<img src="https://img.shields.io/github/languages/code-size/Enchoseon/2based2wait?color=3FA796&labelColor=525E75&style=flat" alt="Code Size">
	<img src="https://img.shields.io/github/package-json/v/Enchoseon/2based2wait?color=FEC260&labelColor=525E75&style=flat" alt="Current Version">
	<img src="https://img.shields.io/github/license/Enchoseon/2based2wait?color=A10035&labelColor=525E75&style=flat" alt="License">
</div>
```
Fix indentation, bold important links, separate badges from important links.

	<a href="https://github.com/Enchoseon/2based2wait/issues/new">Report a Bug</a>
	<span> · </span>
	<a href="https://github.com/Enchoseon/2based2wait/blob/main/docs/configuration-guide.md">Configuration Guide</a>
	<span> · </span>
	<a href="https://github.com/Enchoseon/2based2wait/discussions">Discussions</a>
	</strong>
	<br><br>
	<img src="https://img.shields.io/github/last-commit/Enchoseon/2based2wait?color=2A0944&labelColor=525E75&style=flat" alt="Last Commit">
	<img src="https://img.shields.io/github/languages/code-size/Enchoseon/2based2wait?color=3FA796&labelColor=525E75&style=flat" alt="Code Size">
	<img src="https://img.shields.io/github/package-json/v/Enchoseon/2based2wait?color=FEC260&labelColor=525E75&style=flat" alt="Current Version">
	<img src="https://img.shields.io/github/license/Enchoseon/2based2wait?color=A10035&labelColor=525E75&style=flat" alt="License">
</div>

## Table of Contents

- [Installation](#installation)
- [Features](#features)
- [Images](#images)
- [For Developers](#for-developers)

## Installation

### Prerequisites

- [Git](https://git-scm.com/)
- [NodeJS](https://nodejs.org/)
- [pnpm](https://pnpm.io/): `npm install -g pnpm`

### Quick Start

> 1. Clone the repository: `git clone https://github.com/Enchoseon/2based2wait` and `cd 2based2wait`

Reviewer: ```suggestion
1. Clone the current stable release: `git clone git@github.com:Enchoseon/2based2wait.git --branch v1.0.4` and `cd 2based2wait`
```
Tell users to clone latest stable rather than main.

> 1. Clone the repository: `git clone https://github.com/Enchoseon/2based2wait` and `cd 2based2wait`

Reviewer: ```suggestion
1. Clone the latest release: `git clone https://github.com/Enchoseon/2based2wait --branch v1.0.4 --depth 1 && cd 2based2wait`
```
- Combine two commands into one
  - Imo each command should either be it's own step or combined like this.
- Tell users to shallow clone the (current) latest release
